src/setting_definitions/__init__.py

```diff
@@ -81,6 +81,13 @@ def vault_required(settings):
                               ("fake",       "Do not use passwords from the vault")
                           ]
                           ),
+    EnumSettingDefinition("db_annex_type",
+                          "How do we treat the annex database?",
+                          [
+                              ("fake", "Add a totally safe, but empty, version to the constellation"),
+                              ("readonly", "Read-only access to the real annex"),
+                              ("real", "Full access to the real annex: PRODUCTION ONLY")
+                          ]),
```

Comment:
this moved to solve bug with this setting being referenced in `vault-required`, called below

Reply:
OK, that's also fixed in #63 but I'll deal with the conflict there is this goes in first